src/cudnn/bindgen.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -exu
+
+bindgen \
+  --whitelist-type="^cudnn.*" \
+  --whitelist-var="^cudnn.*" \
+  --whitelist-function="^cudnn.*" \
+  --default-enum-style=rust \
+  --no-doc-comments \
+  --with-derive-default \
+  --with-derive-eq \
+  --with-derive-hash \
+  --with-derive-ord \
+  --size_t-is-usize \
+  --use-core \
+  wrapper.h -- --include-directory "/mnt/e/Max/cudnn/include" --include-directory "/mnt/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/include" \
```

Comment:
```suggestion
  wrapper.h -- -I/usr/local/cuda/include \
```
to keep standard with other bindgens. I guess this doesn't work on windows though?

Reply:
hmm, this is in wsl since bindgen doesn't work on windows for me